SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -438,6 +438,49 @@ \subsection{Advanced Module Configuration}
 displayed (Figure \ref{fig:ConfigITKDefault}); these messages are sorted in
 alphabetical order by module names.
 
+Those who prefer to build ITK using the command line are referred to the online
+cmake command-line tool
+documentation\footnote{\url{https://cmake.org/cmake/help/latest/manual/cmake.1.html}.
+Only some typical use cases are shown here for reference.
+
+\begin{itemize}
+\item \textbf{Example 1}: Build all default modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake [-DITK_BUILD_DEFAULT_MODULES:BOOL=ON]
+      /ITK-build
+\end{minted}
+\normalsize
+
+The default is \code{ITK\_BUILD\_DEFULT\_MODULES=ON}, so it is optional to
+have the \code{-D} argument.
+
+\item \textbf{Example 2}: Enable specific group(s) of modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake -DITKv_BUILD_DEFAULT_MODULES:BOOL=OFF
+      -DBUILD_EXAMPLES:BOOL=OFF
+      -DITKGroup_{Group1}:BOOL=ON
```

Comment:
Since there is already the word `group` in the name of the option, it would be more useful to give a concrete example, e.g. `ITKGroup_Filtering:BOOL=ON` or `ITKGroup_Registration:BOOL=ON`.

Reply:
Done in fddf79c.